lms/djangoapps/certificates/management/commands/regenerate_user.py

```diff
@@ -8,6 +8,7 @@
 from django.core.management.base import BaseCommand, CommandError
 from opaque_keys.edx.keys import CourseKey
 
+from badges.utils import badges_enabled
```

Comment:
Nit: imports should be order alphabetically. This line should come after the following line.